automation/check-patch.e2e-nmstate-functests.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -xeu
+
+# This script should be able to execute Kubernetes-nmstate
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-nmstate-functests.sh
+
+teardown() {
+    rm -rf "${TMP_COMPONENT_PATH}"
+    make cluster-down
+}
+
+main() {
+    # Setup CNAO and artifacts temp directory
+    source automation/check-patch.setup.sh
+    cd ${TMP_PROJECT_PATH}
+
+    # Spin-up ephemeral cluster with latest CNAO
+    # this script also exports KUBECONFIG, and fetch $COMPONENT repository
+    export KUBEVIRT_NUM_NODES=3 # 1 master, 2 workers
+    export KUBEVIRT_NUM_SECONDARY_NICS=2
+    COMPONENT="nmstate" source automation/components-functests.setup.sh
+
+    trap teardown EXIT
+
+    echo "Configure test parameters"
+    export TIMEOUT=${TIMEOUT:-1h}
+    export NAMESPACE=${NAMESPACE:-cluster-network-addons}
```

Comment:
We don't need any of these variables to be configurable. The role of `automation/` scripts is to run on any machine with zero configuration. It should be self sufficient getting its own cluster and its own Go.

Reply:
so you're saying I shouldn't I shouldn't export them?or not even pass these variables to the `make test-e2e-handler` command?

Reply:
Keep exporting them. There is just no need to make them configurable:

Turn `${TIMEOUT:-1h}` to `1h`

Reply:
ack.
[Done](https://github.com/kubevirt/cluster-network-addons-operator/compare/284194c866d30059385fbb563e03094a087cbab8..2c31d617a8d910a39f73ecdfc1c193e4a53bc32e)